WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2666,6 +2667,7 @@ ZmZimbraMail.prototype.setUserInfo = function() {
 		var el = this._userNameField.getHtmlElement();
         el.innerHTML =  AjxStringUtil.htmlEncode(AjxStringUtil.clipByLength(username, 24));
 		el.setAttribute('aria-label', username);
+		el.setAttribute('aria-disabled', true); // Prevent SR to annouch as clickable.
```

Comment:
isn't removing aria attributes the same as aria-disabled?

Reply:
Ignoring the comment as per our offline discussion.